lighthouse-core/formatters/partials/url-list.html

```diff
@@ -1,33 +1,27 @@
 <style>
-  .http-resources {
-    font-size: 14px;
-  }
-  .http-resource__url {
-    margin-right: 8px;
-  }
   .http-resource__protocol,
   .http-resource__code {
-    color: #999;
+    color: var(--secondary-text-color);
   }
   .http-resource__code {
     text-overflow: ellipsis;
     overflow: hidden;
   }
 </style>
 
-<div>
-  <details class="http-resources">
-    <summary>URLs</summary>
-    {{#each this}}
-      <div class="http-resource">
-        <span class="http-resource__url">{{this.url}}</span>
-        {{#if this.label}}
-          <span class="http-resource__protocol">({{this.label}})</span>
-        {{/if}}
-        {{#if this.code}}
-          <pre class="http-resource__code">{{this.code}}</pre>
-        {{/if}}
-      </div>
-    {{/each}}
-  </details>
-</div>
+<details class="subitem__details">
+  <summary class="subitem__detail">URLs</summary>
+  <ul class="subitem__details">
+  {{#each this}}
+    <li class="subitem__detail http-resource">
+      <span class="http-resource__url">{{this.url}}</span>
+      {{#if this.label}}
+        <span class="http-resource__protocol">({{this.label}})</span>
+      {{/if}}
+      {{#if this.code}}
+        <pre class="http-resource__code">{{this.code}}</pre>
```

Comment:
Can reproduce, but can't identify the cause. The extra leading space is within the string (extendedInfo.value.code). That extra space doesn't appear in the original page, though. I tracked it through StyleHelper.getFormattedStyleRule, which doesn't appear to add any extra whitespace. I didn't change the element or the styling applied to that element directly. No Handlebars helpers are being applied to the string. I'm at a loss to what's causing this.